pkg/build/util/generate.go

```diff
@@ -27,9 +27,14 @@ func GenerateBuildFromConfig(bc *buildapi.BuildConfig, r *buildapi.SourceRevisio
 			Revision: r,
 		},
 		ObjectMeta: kapi.ObjectMeta{
-			Labels: map[string]string{buildapi.BuildConfigLabel: bcCopy.Name},
+			Labels: bcCopy.Labels,
```

Comment:
Is the build config thrown away?  What if we start caching build configs and its coming out of the cache?  We already hit this a few times in Kube, so I'm a bit paranoid about it:


> On Feb 20, 2015, at 5:55 PM, Ben Parees <notifications@github.com> wrote:
> 
> In pkg/build/util/generate.go:
> 
> > @@ -27,9 +27,14 @@ func GenerateBuildFromConfig(bc *buildapi.BuildConfig, r *buildapi.SourceRevisio
> >  			Revision: r,
> >  		},
> >  		ObjectMeta: kapi.ObjectMeta{
> > -			Labels: map[string]string{buildapi.BuildConfigLabel: bcCopy.Name},
> > +			Labels: bcCopy.Labels,
> why? the buildconfig is already a copy explicitly made for this purpose...
> 
> —
> Reply to this email directly or view it on GitHub.
>